src/components/Perils/data/usePerils.ts

```diff
@@ -7,7 +7,7 @@ export const usePerils = (insuranceType: TypeOfContract, locale: Locale) => {
   const [perils, setPerils] = useState<[] | Peril[]>([])
 
   const fetchPerils = async () => {
-    const url = `https://giraffe.hedvig.com/graphql`
+    const url = `https://graphql.dev.hedvigit.com/graphql`
```

Comment:
hm. Should we maybe make this an environment variable instead?

Reply:
yes, let's do it when I change it back tomorrow :)